src/hooks/useArticlesInCategory.js

```diff
@@ -28,7 +28,7 @@ const generatePathsMap = (items, acc = {}) => {
   return acc;
 };
 
-export default (category, currentSlug, currentApiVersion) => {
+const useArticlesInCategory = (category, currentSlug, currentApiVersion) => {
```

Comment:
I saw that anonymous functions cause warnings in the console. It's a little bit out of the scope of this PR, but anyway happy to see this :)